src/jvmMain/sass/main.scss

```diff
@@ -1591,6 +1647,36 @@ small.alert-count {
       width: 44px;
     }
   }
+
+  .thin-dd {
+    .dropdown-divider {
+      display: none;
```

Comment:
I'd consider making these visible on larger screens as well, as vertical dividers that is. If you don't agree or want to leave that for another PR, everything else looks good to me.

Reply:
Can leave that for the future